src/components/Overview/__tests__/Overview.test.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Overview } from '../index';
+import { getChildPageNodes } from '@/utils/getChildPageNodes';
+
+const routerMock = {
+  __esModule: true,
+  useRouter: () => {
+    return {
+      query: { platform: 'react' }
+    };
+  }
+};
+
+jest.mock('next/router', () => routerMock);
+
+describe('Overview', () => {
+  const childPageNodes = getChildPageNodes('/[platform]/build-a-backend');
+  const component = <Overview childPageNodes={childPageNodes} />;
+
+  it('should render the Overview component', async () => {
+    render(component);
+    const overviewNode = await screen.getByRole('link', {
+      name: 'Authentication Enable sign-in, sign-up and sign-out within minutes with pre-built UI components and powerful authentication APIs'
+    });
+    const sampleOverviewCard = overviewNode;
+    childPageNodes
```

Comment:
Do we need the ternary here? Or can we just have the `.toBeInTheDocument()` statement? And probably we don't need the `.textContent` check since we are essentially finding the link by that text in the `screen.getByRole` query